article/app/views/liveblog/liveBlogBlocks.scala.html

```diff
@@ -11,7 +11,7 @@
     <div
         id="block-@block.id"
         data-sort-time="@block.publishedCreatedTimestamp()"
-        class="block block--content@block.eventClass"
+        class="block block--content@block.eventClass@block.attributes.membershipPlaceholder.map { _ => js-insert-epic-after}"
```

Comment:
fair enough - though by that reasoning every single `js-*` class would be wrapped in an if statement, which seems overkill. if it doesn't have a harmful effect in an AMP environment then I think I'll leave it in